File: result/translate/starchat/python2java/java/FIND_THREE_ELEMENT_FROM_DIFFERENT_THREE_ARRAYS_SUCH_THAT_THAT_A_B_C_K_1.java
public static boolean f_gold ( int[] a1, int[] a2, int[] a3, int n1, int n2, int n3, int sum ) {
        Set < Integer > s = new HashSet < > ( );
        for ( int i = 0; i < n1; i++ ) s.add ( a1 [ i ] );
        for ( int i = 0; i < n2; i++ ) {
            for ( int j = 0; j < n3; j++ ) {
                if ( sum - a2 [ i ] - a3 [ j ] in s ) return true;
            }
        }
        return false;
    }

File: result/repair/transcoder/java2python/python_repair/FIND_THREE_ELEMENT_FROM_DIFFERENT_THREE_ARRAYS_SUCH_THAT_THAT_A_B_C_K_1.py
def f_gold ( a1 , a2 , a3 , n1 , n2 , n3 , sum ) :
    s = set ( )
    for i in range ( n1 ) :
        s.add ( a1 [ i ] )
    al = [ i for i in range ( n2 ) if i != a2 [ i ] ]
    i=0 
    while i<n2 :
        for j in range ( n3 ) :
            if al.count ( sum - a2 [ i ] - a3 [ j ] ) & al.count ( sum - a2 [ i ] - a3 [ j ] ) != al [ - 1 ] :
                return True
        i += 1
    return False

